### PR TITLE
supermatter spiders! (funny admin mob)

### PR DESCRIPTION
**Pull request by capsaicinz**
## About The Pull Request

adds little creachures that will aggressively hunt down a target and dust it, killing itself in the process. tend to enjoy hunting in swarms, for sport mostly.

unknown

## Why It's Good For The Game

really just a cool mob for admins to send down upon non-belivers.

(yes i know my master is fucked stop talking about it i beg you)

## Changelog
:cl:
add: Adds supermatter spiders.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
CI's failed, looks like it's trying to give cult constructs the sm spider icon state? if you want help fixing it bug me yeah?

File: code/modules/mob/living/simple_animal/hostile/smspider.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/mob/living/simple_animal/hostile
```
Comment (Mothblocks) [inline code/modules/mob/living/simple_animal/hostile/smspider.dm:1]: Bruh

Comment (capsaicinz) [reply code/modules/mob/living/simple_animal/hostile/smspider.dm]: get out of my head

**Comment by tommysalami3** [comment]
A Dubious Little Creature... Up To No Good

**Comment by thgvr** [comment]
will xenobio gold extracts spawn these?

**Comment by IndieanaJones** [comment]
I would make the changelog different seeing as what it claims isn't actually backed by the code.  Might encourage to try and make them when spiders hitting the SM doesn't actually do anything.

**Comment by capsaicinz** [comment]
> will xenobio gold extracts spawn these?

no they shouldn't

**Pull request merged**